### ISSUE TITLE
adding ISF to a graph - Feature Request

### ISSUE THREAD

**Issue by ohjay73** [zreptil/nightscout-reporter#74]
With the addition of DynISF and AutoISF would it be possible to pull the profile ISF and variable ISF in a graph compared to each other as well as the existing BG? All of that data appears in Nightscout. 

I couldnt get "feedback" to work on the webpage so added the req here.

**Comment by zreptil** [comment]
Hi ohjay73,

how should this look in a graph so that you can have some valuable information from it? if you can tell me, what would help you to analyze the data and give me the url to your nightscout instance, i can take a look, to see, if i can make a graph that shows, what you want to see.

**Comment by ohjay73** [comment]
tbh.. this tool does it well, but is missing profile BG.. Just thought it
would be helpful to see the delta in your reports as they goto the doctors
in many cases.

If you dont have a NS that is using DynISF, you can use mine, just would
rather not post my NS on a public github.

https://rafaeldelrey-nsview-nsview-9csanz.streamlit.app/


[image: image.png]

**Comment by zreptil** [comment]
Then I still need the url to your NS, since I don't have the data. You can mail it to me at [nightscoutreporter@gmail.com](mailto:nightscoutreporter@gmail.com). This is what the Feedback-Menuentry does. It simply calls mailto:. What happens, when you try to click on feedback in the menu? It should open a new tab with mailto:nightscoutreporter@gmail.com along with some parameters that should be handled by your browser.

**Comment by ohjay73** [comment]
Did my email make it to you?

**Comment by zreptil** [comment]
Yes, it reached me. But up to now I had no time to look into it. I have to understand what is going on there and see, what Nightscout Reporter can do for you. :)

**Comment by ohjay73** [comment]
No worries.. just thought it went to vaporland.. thanks for replying to
confirm it make it through
